config/__init__.py

```diff
@@ -2,13 +2,23 @@
 
 FLASK_ENV = environ.get("FLASK_ENV")
 
-if FLASK_ENV == "development":
-    from config.environments.development import DevelopmentConfig as Config
-elif FLASK_ENV == "dev":
-    from config.environments.dev import DevConfig as Config
-elif FLASK_ENV == "test":
-    from config.environments.test import TestConfig as Config
-elif FLASK_ENV == "production":
-    from config.environments.production import ProductionConfig as Config
-else:
-    from config.environments.default import Config  # noqa
+match FLASK_ENV:
+    case "development":
+        from config.environments.development import (
+            DevelopmentConfig as Config,
+        )  # noqa
+    case "dev":
+        pass
+    case "test":
+        from config.environments.test import TestConfig as Config  # noqa
+    case "production":
+        pass
+    case _:
+        from config.environments.default import DefaultConfig as Config  # noqa
```

Comment:
🔧 Add unit_test and dev a la discovery frontend.

Reply:
👍